libraries/azure-app-configuration-importer/src/internal/parsers/defaultConfigurationSettingsConverter.ts

```diff
@@ -157,27 +164,65 @@ class FeatureFlagConfigurationSettingsConverter implements ConfigurationSettings
   Convert(
     config: object,
     options: SourceOptions
-  ): SetConfigurationSettingParam<FeatureFlagValue>[] {
-    const settings = new Array<SetConfigurationSettingParam<FeatureFlagValue>>();
+  ): SetConfigurationSettingParam[] {
+    const settings = new Array<SetConfigurationSettingParam>();
 
-    for (const featureFlag in config) {
-      if (!this.validateFeatureName(featureFlag)) {
+    if (this.checkIfUsingFFV2Schema(config)) {
```

Comment:
As what we discussed, the new schema and old schema is not exclusive from each other, they may coexist. So we can just 
```
if (newSchema){
}
else {
}
```